archeo/templates/single.html

```diff
@@ -51,17 +51,23 @@
 
 	<!-- wp:columns {"isStackedOnMobile":false,"align":"wide","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} -->
 		<div class="wp-block-columns alignwide is-not-stacked-on-mobile" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:column -->
-		<div class="wp-block-column"><!-- wp:separator {"className":"is-style-wide"} -->
-		<hr class="wp-block-separator is-style-wide"/>
+		<div class="wp-block-column">
+			
+		<!-- wp:separator {"style":{"color":{"background":"#fcfbf766"}},"className":"is-style-wide"} -->
+		<hr class="wp-block-separator has-text-color has-alpha-channel-opacity has-background is-style-wide" style="background-color:#fcfbf766;color:#fcfbf766"/>
 		<!-- /wp:separator -->
+
 		<!-- wp:post-navigation-link {"type":"previous","label":"Previous Post"} /-->
 		</div>
 		<!-- /wp:column -->
 
 		<!-- wp:column -->
-		<div class="wp-block-column"><!-- wp:separator {"className":"is-style-wide"} -->
-		<hr class="wp-block-separator is-style-wide"/>
+		<div class="wp-block-column">
+		
+		<!-- wp:separator {"style":{"color":{"background":"#fcfbf766"}},"className":"is-style-wide"} -->
+		<hr class="wp-block-separator has-text-color has-alpha-channel-opacity has-background is-style-wide" style="background-color:#fcfbf766;color:#fcfbf766"/>
```

Comment:
This isn't one of the palette colors.  Can it be?

Reply:
That is the palette color named `background` (`#FCFBF8`) with the alpha channel set to 40% to preserve the same look. 

If we use `color.text` instead of `color.background` in `theme.json` we will need this change. The rationale:

We need to add that because otherwise, these separators that get their color using `currentColor` become invisible because the block color set in `theme.json` is `foreground` (kind of black). `currentColor` only works with `color` (`color.text` in `theme.json`) that's why is impacted by the change in theme.json and why we need to set a color in this separator.

The CSS classes are added by Gutenberg to handle the legacy separator block version. I didn't add markup manually it was just the output of the editor.